crates/viewer/re_viewer/src/web.rs

```diff
@@ -22,6 +25,11 @@ use crate::web_tools::{
 static GLOBAL: AccountingAllocator<std::alloc::System> =
     AccountingAllocator::new(std::alloc::System);
 
+struct Channel {
+    log_tx: re_smart_channel::Sender<re_log_types::LogMsg>,
+    table_tx: crossbeam::channel::Sender<re_log_types::TableMsg>,
```

Comment:
I guess a consequence of not putting `TableMsg` as a variant of `LogMsg` is that you cannot store tables in .rrd files. Is this intentional?

Reply:
Yes, we specifically said that tables shouldn't end up in `.rrd` files, because there are much better formats for them.

Reply:
That decision should go into the docstring of `TableMsg` and/or `LogMsg` then